### PR TITLE
Update Conda tuto

### PR DESCRIPTION
**Pull request by abretaud**
So here's a few changes to the conda recipe tutorial, to make it more clear and less potentially dangerous for bioconda github repo

### CODE REVIEW

File: topics/dev/tutorials/tool-from-scratch/tutorial.md [hunk 1]
```diff
@@ -71,21 +71,32 @@ Recipes should always define the following 6 sections in the `meta.yaml` file:
 - test
 - about
 
+Let's write a Bioconda recipe for the tool we want to package: [bellerophon](https://github.com/davebx/bellerophon)
+
+> ### {% icon warning %} Naming collision
+> As this tool is already packaged in Bioconda, to prevent any naming collision, we will slightly modify its name from `bellerophon` to `bellerophon_bis` in the recipe.
+> This should never be done in real life, we only do it for this exercise!
```
Comment (hexylena) [inline topics/dev/tutorials/tool-from-scratch/tutorial.md:78]:
```suggestion
> **This should never be done in real life, we only do it for this exercise!**
```

File: topics/dev/tutorials/tool-from-scratch/tutorial.md [hunk 2]
```diff
@@ -71,21 +71,32 @@ Recipes should always define the following 6 sections in the `meta.yaml` file:
...
+{: .warning}
+
 > ### {% icon hands_on %} Hands-on: Writing a Bioconda Recipe
 >
-> 1. The first thing we should do is prepare our workspace. We create a branch in git, a conda environment with pip and
+> 1. The first thing we should do is prepare our workspace. [Create a fork](https://help.github.com/articles/fork-a-repo/) of the [Bioconda repository](https://github.com/bioconda/bioconda-recipes/) on GitHub
+>
+> 2. Clone your fork of this repository to create a local copy on your computer
```
Comment (hexylena) [inline topics/dev/tutorials/tool-from-scratch/tutorial.md:85]: Ok, I really need to get to writing more git tutorials...

File: topics/dev/tutorials/tool-from-scratch/tutorial.md [hunk 3]
```diff
@@ -147,8 +158,8 @@ Recipes should always define the following 6 sections in the `meta.yaml` file:
 >
 >     {% endraw %}
 >
-> 5. Next, we move on to the build metadata. Since this is the first version of the conda recipe, the build number is 0.
-> We use the externally defined `{{ PYTHON }}` variable, which defines which python conda-build is using, to install it
+> 8. Next, we move on to the build metadata. Since this is the first version of the conda recipe, the build number is 0.
+> We use the externally defined `{% raw %}{{ PYTHON }}{% endraw %}` variable, which defines which python conda-build is using, to install it
```
Comment (hexylena) [inline topics/dev/tutorials/tool-from-scratch/tutorial.md:162]: :fearful:

**Review by hexylena — APPROVED** [review]
This is a great improvement @abretaud

File: topics/dev/tutorials/tool-from-scratch/tutorial.md (hunk 3 above)
Comment (abretaud) [reply topics/dev/tutorials/tool-from-scratch/tutorial.md:162]: Yep, it was not displayed, not sure if there's a simpler syntax trick?

Comment (hexylena) [reply topics/dev/tutorials/tool-from-scratch/tutorial.md:162]: No, that's right, I'm just horrified we missed it

Comment (abretaud) [reply topics/dev/tutorials/tool-from-scratch/tutorial.md:162]: Yeah it's easy to miss this kind of problem when reviewing

Comment (hexylena) [reply topics/dev/tutorials/tool-from-scratch/tutorial.md:162]: https://jekyllrb.com/docs/configuration/liquid/ looks like we might be able to set a stricter liquid setting and catch those. I'll test it out (unrelated to the PR)

**Comment by hexylena** [comment]
Merging over the failure because someone's over their account limit for some node package in azure.

**Review by hexylena — APPROVED** [review]

**Comment by abretaud** [comment]
Oops, deploy failed

**Comment by hexylena** [comment]
hopefully fixed, so many `ffi` issues all of a sudden.

**Pull request merged**